### ISSUE TITLE
Salt-bootstrap require dirmngr

### ISSUE THREAD

**Issue by drouarb** [saltstack/salt-bootstrap#1109]
### Description of Issue/Question
Salt bootstrap require dirmngr but don't install it. Work fine if you install it manually.

### Setup
Install a fresh debian 9.

### Steps to Reproduce Issue
```
gpg: requesting key from 'https://repo.saltstack.com/apt/debian/8/amd64/latest/SALTSTACK-GPG-KEY.pub'
gpg: failed to start the dirmngr '/usr/bin/dirmngr': Aucun fichier ou dossier de ce type
gpg: connecting dirmngr at '/tmp/apt-key-gpghome.rgJCyrRIWX/S.dirmngr' failed: Aucun fichier ou dossier de ce type
gpg: WARNING: unable to fetch URI https://repo.saltstack.com/apt/debian/8/amd64/latest/SALTSTACK-GPG-KEY.pub: Pas de dirmngr
 * ERROR: Failed to run install_debian_deps()!!!
```

### Versions and Systems
./bootstrap-salt.sh -- Version 2017.05.24

**Comment by rallytime** [comment]
@drouarb Thanks for filing this issue. I am adding support for Debian 9 right now since we have official packages in `repo.saltstack.com` for Debain 9, so I will take a look.

**Comment by rallytime** [comment]
Hi @drouarb - I have fixed this in pull request #1111.
